hca/util/__init__.py

```diff
@@ -251,8 +251,9 @@ def swagger_spec(self):
                 res = self.get_session().get(swagger_url)
                 res.raise_for_status()
                 assert "swagger" in res.json()
-                with open(swagger_filename, "wb") as fh:
-                    fh.write(res.content)
+                self._atomic_write_to_file(self._get_path_leaf(swagger_filename),
```

Comment:
os.path.basename doesn't work here?

Reply:
Oh, it would - I was covering an edge case that, actually, we don't need to worry about: https://stackoverflow.com/questions/8384737/extract-file-name-from-path-no-matter-what-the-os-path-format
-> Will replace `ntpath.basename` with `os.path.basename`

However, `_get_path_leaf` handles the trailing slash case (e.g. `path/to/file/`).